doc/dev/crimson/crimson.rst

```diff
@@ -146,6 +125,106 @@ Running Crimson
 
 As you might expect, Crimson does not yet have as extensive a feature set as does ``ceph-osd``.
 
+vstart.sh
+=========
+
+The following options can be used with ``vstart.sh``.
+
+``--crimson``
+    Start ``crimson-osd`` instead of ``ceph-osd``.
+
+``--nodaemon``
+    Do not daemonize the service.
+
+``--redirect-output``
+    Redirect the ``stdout`` and ``stderr`` to ``out/$type.$num.stdout``.
+
+``--osd-args``
+    Pass extra command line options to ``crimson-osd`` or ``ceph-osd``.
+    This is useful for passing Seastar options to ``crimson-osd``. For
+    example, one can supply ``--osd-args "--memory 2G"`` to set the amount of
+    memory to use. Please refer to the output of::
+
+      crimson-osd --help-seastar
+
+    for additional Seastar-specific command line options.
+
+``--crimson-smp``
+    The number of cores to use for each OSD.
+    If BlueStore is used, the balance of available cores
+    (as determined by `nproc`) will be assigned to the object store.
+
+``--bluestore``
+    Use the alienized BlueStore as the object store backend. This is the default (see below section on the `object store backend`_ for more details)
```

Comment:
A little bit of sneaky editing here; I've re-arranged the arguments so that the object store types are all together, and also I've put vstart before the section about the object store backend for reasons mentioned in the comment above, so I've included a link to the chapter here.

Reply:
We have a convention in docs of writing "back end" as two words.  I personally prefer one, but such is the party line.  Here and below.